.github/workflows/ci-update-workflow.yml

```diff
@@ -0,0 +1,63 @@
+name: ci-update-workflow
+
+on:
+  schedule:
+  # Run every hour.
```

Comment:
I'm wondering if we really need it to be every hour. Could you share more context on this?

Reply:
I configured it to run every hour just because it's lightweight so it doesn't hurt to run it more often. I changed it to "every day at 10AM" if it looks better to you.